.github/workflows/eventmesh-server-go.yml

```diff
@@ -1,52 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-#
-
-name: "Continuous Integration of EventMesh Server Go"
-
-on:
-  push:
-    branches: ['*']
-  pull_request:
-    branches: ['*']
-
-jobs:
-  build:
-    name: Build
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - uses: actions/setup-go@v3
-        with:
-          go-version-file: 'eventmesh-server-go/go.mod'
-      - run: go version
-
-      - name: Download components
-        run: go mod tidy
-        working-directory: eventmesh-server-go
-
-      - name: Build application
-        run: make build
-        working-directory: eventmesh-server-go
-
-      - name: Test application
-        run: make test
-        working-directory: eventmesh-server-go
```

Comment:
Why should this file be deleted? Already have an alternate function?

Reply:
I found `eventmesh-server-go` has been deleted from the newest master branch. And it will fail due to not found the folder/files.

Reply:
Ok, I'm going to sync the warehouse.